docs/release-notes.md

```diff
@@ -1,3 +1,11 @@
+## 0.20.2
+
+June 4, 2022
+
+### Fixed
+* Fix regression on route paths with colons [#1675](https://github.com/encode/starlette/pull/1675).
+* Allow staticfiles to follow symlinks [#1337](https://github.com/encode/starlette/pull/1377).
```

Comment:
Is this an addition or really a fix? I don't care. Just questioning in case.

Reply:
Yeah it's a bit hard to tell, internally it was designed this way, but from the issue this is introduced as a bug. Open to suggestion though.

Reply:
Your call. Do as you wish.